services/app/src/pages/game/Game.jsx

```diff
@@ -89,15 +89,14 @@ function Room() {
       <button disabled={finished || !canPlay} type="button" onClick={finishMove}>
         submit
       </button>
-      <button type="button" disabled={!canPlay} onClick={clearBoard}>
-        Clear Board
-      </button>
       <div className="game-content">
         <Team
           playerCount={players.length}
           players={team}
           skipMove={skipMove}
           removePlayer={removePlayer}
+          canPlay={canPlay}
```

Comment:
`canPlay` and `clearBoard` are passed from `ChessBoardContext` which is used in the `Team` component. Why not grab those props from inside the `Team` component instead of passing it down, as you're already using `ChessBoardContext` inside the `Team`?